### ISSUE TITLE
Require date entry in ISO 8601 standard (YYYY-MM-DD)

### ISSUE THREAD

**Issue by juliaxbow** [pln-planning-tools/Starmap#275]
Dates not following this format should return an error message in the starmap error log. This should be communicated out first and roadmap maintainers given an opportunity to update their roadmaps if necessary

**Comment by momack2** [comment]
We used to support "2023Q2" - do we need to update all issues that still use this format? cc @SgtPooki

**Comment by momack2** [comment]
I'm also seeing conflicting results - ex https://github.com/libp2p/go-libp2p/issues/2034 seems like it does hit the ISO standard - but I see this error: 
<img width="983" alt="CleanShot 2023-05-15 at 20 49 17@2x" src="https://github.com/pln-planning-tools/Starmap/assets/618519/7bfcca3e-fa97-4ca6-a098-086e17a4d236">
<img width="429" alt="CleanShot 2023-05-15 at 20 49 52@2x" src="https://github.com/pln-planning-tools/Starmap/assets/618519/8ef8cb6f-0b7c-4585-a830-f35c63a6187f">

**Comment by SgtPooki** [comment]
@momack2 yes we need to update all ETAs to support the full iso format. I believe this is a request you were pushing for.

I will look into that issue, might be a caching problem.
